api/src/pcapi/admin/base_configuration.py

```diff
@@ -72,26 +73,15 @@ def is_accessible(self) -> bool:
         return authorized
 
 
-class FlaskWTFSecureForm(SecureForm):
+class FlaskWTFSecureForm(BaseForm, FlaskForm):
     """
-    To be used only if Flask-WTF is activated: it will handle all the
-    CSRF protection. Both FlaskAdmin internal configuration for CSRF
-    protection and Flask-WTF cannot work together.
-
-    The later adds some defaults (config and jinja filters) that will
-    automatically add a CSRF token and run all the related security
-    checks.
-
-    Note:
-        Flask-WTF has an `exempt()` method that deactivates CSRF
-        protection for a blueprint... which sounds great but the
-        overriden defaults stays. The result is that a FlaskAdmin view
-        with a SecureForm will have two CSRF tokens... and the form will
-        be considered invalid.
+    FlaskForm handles all the CSRF protection.
+
+    BaseForm is needed because FlaskAdmin expects forms to inherit from
+    this base class.
     """
 
-    class Meta:
-        csrf = False
+    pass  # pylint: disable=unnecessary-pass
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏pylint rale si tu enlève le `pass` ?

Reply:
Oui :(

```
************* Module pcapi.admin.base_configuration
src/pcapi/admin/base_configuration.py:83: [W0107(unnecessary-pass), FlaskWTFSecureForm] Unnecessary pass statement
```

Reply:
@jeremieb-pass : @lixxday proposait d'enlever le `pass` lui-même : 

```suggestion
```

Reply:
oula oui désolé. Trop tard.